pkg/platform/kube/controller/nucliofunction.go

```diff
@@ -90,6 +91,20 @@ func (fo *functionOperator) CreateOrUpdate(ctx context.Context, object runtime.O
 			errors.New("Received unexpected object, expected function"))
 	}
 
+	defer common.CatchAndLogPanicWithOptions(ctx, // nolint: errcheck
```

Comment:
Add it to Delete as well no ?

Reply:
tl;dr: no

since the flow is currently limited to how it updates the function status (it requires the object as oppose to delete requires object name), and the fact that panic is always recovered on the worker level, regardless, I will leave it as is for now.